.github/ISSUE_TEMPLATE/failure_building_from_source.yml

```diff
@@ -17,9 +17,8 @@ body:
           - **OS**: Ubuntu 20.04
           - Sage Version: 9.2
       value: |
-          - **OS**:
+          - **OS**: 
           - **Sage Version**: 
```

Comment:
seems an extra space is added in L20.

Reply:
That was intentional, there should be a space between the colon and the OS name. This just saves the report submitter from having to add the space themselves.

We want

- **OS**: Ubuntu 20.04

not

- **OS**:Ubuntu 20.04

Reply:
Ah, right. I thought only that line has a space, inconsistently.

OK.